src/cloud/live/sens-data-over.md

```diff
@@ -60,6 +62,17 @@ All system configurations are set during build and deploy according to the follo
 1. If an environment variable does not exist, use the configuration from a `MAGENTO_CLOUD_RELATIONSHIPS` name-value pair in the [`.magento.app.yaml` file][app-yaml]. Ignore the default configuration.
 1. If an environment variable does not exist and `MAGENTO_CLOUD_RELATIONSHIPS` does not contain a name-value pair, remove all customized configuration and use the values from the default configuration.
 
+If the same setting is configured in multiple places, Magento relies on the following configuration hierarchy to determine which value to apply to the environment :
+
+|Priority|Configuration<br/>Method|Description|
+|---|---|---|
```

Comment:
This table seems to have an unnatural order. My spidey-sense wants to see priority listed in some sequential order. Is there a method you used that I overlooked or can this be reordered?

Reply:
The strange order seems to be something that happened in my VS code instance.  I'm not sure what happened. Should be in sequential order.